sdk/eventgrid/azure-messaging-eventgrid-cloudnative-cloudevents/src/samples/java/com.azure.messaging.eventgrid.cloudnative.cloudevents.samples/PublishNativeCloudEvent.java

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
The directory name looks odd. `com.azure.messaging...` should be `com/azure/messaging/...`.
`sdk/eventgrid/azure-messaging-eventgrid-cloudnative-cloudevents/src/samples/java/com.azure.messaging.eventgrid.cloudnative.cloudevents.samples/PublishNativeCloudEvent.java`